packages/terra-collapse/tests/nightwatch/CustomizedButtonCollapse.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import Collapse from '../../lib/Collapse';
+
+const ButtonAttrsCollapse = () => (
+  <Collapse closedButtonText="Collapse" buttonAttrs={{ variant: 'link' }}>
```

Comment:
`buttonAttrs` can be used to set both the `props` and `html attrs` on the `<Button />` component.